src/assets/translations/en/main.json

```diff
@@ -2782,6 +2782,14 @@
       "totalStakingValue": "Total Staking Value",
       "nextEpoch": "Next Epoch"
     },
+    "foxWifHat": {
+      "title": "FOX Wif Hat",
+      "description": "It's just FOX wif a hat, don't overthink it. Enjoy a limited time onchain discount program. As your discount decays learn about FOX on the rest of the page and swap into FOX any time to preserve your fee discount. Whichever holding is higher saves you money on the Shapeshift platform.",
```

Comment:
```suggestion
      "description": "It's just FOX wif a hat, don't overthink it. Enjoy a limited time on-chain discount program. As your discount decays learn about FOX on the rest of the page and swap into FOX any time to preserve your fee discount. Whichever holding is higher saves you money on the ShapeShift platform.",
```

Reply:
Nice catch! Done in [2e3a203](https://github.com/shapeshift/web/pull/8605/commits/2e3a203909b4eca964e0f8520208a6f5cc7c099c)